S2VX.Game.Tests/HeadlessTests/EditorNoteTests.cs

```diff
@@ -20,7 +30,8 @@ public class EditorNoteTests : S2VXTestScene {
         private void Load() {
             StoryClock = new StopwatchClock();
             Story.Clock = new FramedClock(StoryClock);
-            Add(Story);
+            var audioPath = Path.Combine("TestTracks", "10-seconds-of-silence.mp3");
+            Add(new ScreenStack(Editor = new EditorScreen(Story, S2VXTrack.Open(audioPath, Audio))));
```

Comment:
You can add the Editor without assigning it

```suggestion
            Add(new ScreenStack(new EditorScreen(Story, S2VXTrack.Open(audioPath, Audio))));
```

Reply:
Oh ok. Fixed in the latest commit